### ISSUE TITLE
Using image vs fly.toml for launching app

### ISSUE THREAD

**Issue by SimonLab** [dwyl/gogs-server#1]
On the [create an app](https://github.com/dwyl/gogs-server#create-an-app) section we have the following command:

```sh
flyctl launch --name gogs-server --image gogs/gogs --org dwyl
```
This will create a new Fly app with the gogs image, (I guess from https://hub.docker.com/r/gogs/gogs).
However the `fly.toml`also contains in the build section the gogs image:
https://github.com/dwyl/gogs-server/blob/5e434c77823de8240e761ea938cf4d16c57870cb/fly.toml#L12-L13

So could we instead of using the `--image gogs/gogs` flag, just run

```sh
flyctl launch --name gogs-server --org dwyl
```
We could also not pass the `--org` flag and let the command line ask later on the launch step I think.

**Comment by nelsonic** [comment]
@SimonLab yeah, I was just building it from first principals as if the `fly.toml` file didn't exist. ✍️ 
But yeah, we could update the docs to the command you suggest now that the file is there. 💭
